lib/workers/repository/update/pr/index.ts

```diff
@@ -57,6 +63,18 @@ export type ResultWithoutPr = {
 
 export type EnsurePrResult = ResultWithPr | ResultWithoutPr;
 
+export function updatePrDebugData(
+  debugData: PrDebugData | undefined
+): PrDebugData {
+  const createdByRenovateVersion =
+    debugData?.createdByRenovateVersion ?? pkg.version;
```

Comment:
Let's consider all the PRs which exist today without any debug data. Will this update them to say they were created with the current version of Renovate? I think I'd prefer to leave them empty/unknown which is more accurate than adding the current version.

Reply:
yes, it will, that is what @zharinov suggested above, in a week this wouldn't matter anymore, so i listened to him, 
in order to know if the debugData in the PR existed or not  and then if it's an existing PR or a New PR requires lots of computations, my code did that before, but zharinov suggested to remove the extra complications so i removed it. 

i dont think that's a problem, we will know the false data in the first 1-2 weeks of PRs being opened cuz of their creation date, we will believe only PRs that were created after a certain date.
it's better this way, less accurate at start, but cleaner code and good for the future